ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallDataExtensions.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import android.util.Log
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.core.os.ConfigurationCompat
+import androidx.core.os.LocaleListCompat
+import com.revenuecat.purchases.paywalls.PaywallData
+import java.util.Locale
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.getColors(): PaywallData.Configuration.Colors {
+    return config.colors.dark?.takeIf { isSystemInDarkTheme() } ?: config.colors.light
+}
+
+@Composable
+@ReadOnlyComposable
+internal fun PaywallData.localizedConfig(): PaywallData.LocalizedConfiguration? {
+    val locale = getLocale()
+    val configForLocale = configForLocale(locale)
+    if (configForLocale == null) {
+        Log.e("RevenueCatUI", "No configuration found for locale $locale")
+    }
+    return configForLocale
+}
+
+@Composable
+@ReadOnlyComposable
+private fun getLocale(): Locale {
+    val configuration = LocalConfiguration.current
+    return ConfigurationCompat.getLocales(configuration).get(0) ?: LocaleListCompat.getDefault()[0] ?: Locale.ENGLISH
```

Comment:
Nice, so there is a list of configured locales, that's great! We can do the same we did an iOS and find the first one that's available:
https://github.com/RevenueCat/purchases-ios/blob/paywalls/Sources/Paywalls/PaywallData%2BLocalization.swift#L21-L26

Reply:
Just added a Liner for this.